smap/sources/aqiCalculator.py

```diff
@@ -0,0 +1,97 @@
+import requests
+import simplejson as json
+
+# used for logging
+import logging
+import sys, traceback
+logging.basicConfig(level=logging.DEBUG, format='%(asctime)s - %(levelname)s - %(message)s')
+
+BASE_URL = "http://localhost/raqmn_api/"
+
+def init_aqi_drivers():
```

Comment:
This function gets the data of all the Dylos Sensors and is used to initialize the AQI Driver